conmon/conmon.c

```diff
@@ -884,7 +884,8 @@ static void write_sync_fd(int sync_pipe_fd, int res, const char *message)
 static char *setup_console_socket(void)
 {
 	struct sockaddr_un addr = {0};
-	char csname[PATH_MAX] = "/tmp/conmon-term.XXXXXXXX";
+	_cleanup_free_ const char *tmpdir = g_get_tmp_dir();
+	_cleanup_free_ char *csname = g_build_filename(tmpdir, "conmon-term.XXXXXX", NULL);
```

Comment:
According to this page:  https://developer.gnome.org/glib/stable/glib-Miscellaneous-Utility-Functions.html#g-get-tmp-dir
g_get_tmp_dir() can return "/tmp".  Do we need to ad a slash before conmon-term here or am I looking at the wrong function or bad doc?